isis/src/control/objs/BundleUtilities/unitTest.cpp

```diff
@@ -503,9 +503,9 @@ int main(int argc, char *argv[]) {
     // bo3.initializeBodyRotation(); //Seg fault
 
     qDebug() << "    output bundle observation...";
-    qDebug().noquote() << bo3.formatBundleOutputString(false,true);
-    qDebug().noquote() << bo3.formatBundleOutputString(false);
-    qDebug().noquote() << bo3.formatBundleOutputString(true,true);
+    qDebug().noquote() << bo3.bundleOutputCSV(false);
+    //qDebug().noquote() << bo3.formatBundleOutputString(false);
```

Comment:
Another comment to delete, ditto to the rest of the commented out lines in this file.